database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Alter Down/10_DML_PIMS_STATIC_VARIABLE_VERSION.sql

```diff
@@ -0,0 +1,26 @@
+/* -----------------------------------------------------------------------------
+Update the database version in the PIMS_STATIC_VARIABLE table.
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2021-Nov-24  Initial version
+----------------------------------------------------------------------------- */
+
+BEGIN TRANSACTION;
+
+DECLARE @CurrVer NVARCHAR(100)
+SET @CurrVer = N'33.00'
```

Comment:
@FuriousLlama if we are treating s34 as its own database version this should be changed to 34

Reply:
Well this is the Alter Down, so it still should set the DB to 33